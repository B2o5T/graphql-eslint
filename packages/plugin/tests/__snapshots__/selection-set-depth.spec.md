// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
⚙️ Options

    {
      "maxDepth": 1
    }

❌ Error

      1 |         query deep2 {
      2 |           viewer {
      3 |             albums {
    > 4 |               title
        |               ^ 'deep2' exceeds maximum operation depth of 1
      5 |             }
      6 |           }
      7 |         }
`;

exports[` 2`] = `
⚙️ Options

    {
      "maxDepth": 1
    }

❌ Error

      1 |         query deep2 {
      2 |           viewer {
      3 |             albums {
    > 4 |               ...AlbumFields
        |               ^ 'deep2' exceeds maximum operation depth of 1
      5 |             }
      6 |           }
      7 |         }
`;
