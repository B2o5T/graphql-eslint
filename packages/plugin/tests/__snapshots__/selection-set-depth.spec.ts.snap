// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
  1 |
  2 |         query deep2 {
  3 |           viewer {
  4 |             albums {
> 5 |               title
    |                ^ 'deep2' exceeds maximum operation depth of 1
  6 |             }
  7 |           }
  8 |         }
  9 |       
`;

exports[` 2`] = `
  1 |
  2 |         query deep2 {
  3 |           viewer {
  4 |             albums {
> 5 |               ...AlbumFields
    |                ^ 'deep2' exceeds maximum operation depth of 1
  6 |             }
  7 |           }
  8 |         }
  9 |       
`;
